### PR TITLE
fix webapp test

### PR DESCRIPTION
**Pull request by yungezz**
##### SUMMARY

Fix webapp test, https://github.com/VSChina/ansible-issue-tracker/issues/524
1. failed to get key 'id' when create app service plan. Use explicit object to dict instead of as_dict
2. remove unstable tag

##### ISSUE TYPE

- Bugfix Pull Request

##### ANSIBLE VERSION

```paste below
ansible 2.8
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: packaging/requirements/requirements-azure.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ azure-cli-core==2.0.35
 azure-cli-nspkg==3.0.2
 azure-common==1.1.11
 azure-mgmt-batch==4.1.0
-azure.mgmt.cdn==3.0.0
+azure-mgmt-cdn==3.0.0
```
Comment (zikalino) [inline packaging/requirements/requirements-azure.txt:7]: so is it really necessary? as it somehow worked with broken requirement....

**Review by zikalino — APPROVED** [review]

File: packaging/requirements/requirements-azure.txt (hunk 1 above)
Comment (yungezz) [reply packaging/requirements/requirements-azure.txt:7]: i think it's necessary to do right thing. actually i didn't understand why previous version works :(

Comment (zikalino) [reply packaging/requirements/requirements-azure.txt:7]: so are you sure we need this dependency?

Comment (zikalino) [reply packaging/requirements/requirements-azure.txt:7]: oh, perhaps correct version is in test requirements..... but then sanity tests wouldn't pass....

Comment (zikalino) [reply packaging/requirements/requirements-azure.txt:7]: yes, I think you also need to update in test/runner/requirements

Comment (yungezz) [reply packaging/requirements/requirements-azure.txt:7]: nice catch. fixed. the dependency is for cdn.

**Pull request merged**